src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -492,11 +501,17 @@ public function productVariantBelongsToTaxCategory(
 
     /**
      * @Given /^(this product) has ([^"]+) attribute "([^"]+)" with value "([^"]+)"$/
+     * @Given /^(this product) has ([^"]+) attribute "([^"]+)" with value "([^"]+)" in ("[^"]+" locale)$/
```

Comment:
Matching every character except `"` does not make any sense here, let's match all the characters instead.